launch.py

```diff
@@ -14,7 +14,7 @@
 args, _ = parser.parse_known_args(sys.argv)
 
 script_path = os.path.dirname(__file__)
-data_path = os.getcwd()
+data_path = args.data_dir

```

Comment:
```suggestion
data_path = args.data_dir or os.getcwd()
```
shouldn't this be `or`'d in order to default to the old behavior when the argument is not specified?

Reply:
Since `default` is specified below, the base directory is assigned when the argument is not specified.

https://github.com/AUTOMATIC1111/stable-diffusion-webui/blob/b9a66b02d0b8ca1c3364d9410198530682fd2f6c/launch.py#L13